barretenberg/cpp/src/barretenberg/benchmark/ipa_bench/ipa.bench.cpp

```diff
@@ -3,28 +3,22 @@
 
 using namespace benchmark;
 using namespace bb;
-using namespace bb::honk::pcs::ipa;
+
 namespace {
 using Curve = curve::Grumpkin;
 using Fr = Curve::ScalarField;
-using IPA = IPA<Curve>;
-using OpeningPair = honk::pcs::OpeningPair<Curve>;
```

Comment:
With the names much less unwieldy, I'd like to get away from these `using` statements in favour of just writing out the template instance